extensions/bucket-docs/src/utils/index.ts

```diff
@@ -0,0 +1,79 @@
+import { load } from "cheerio";
```

Comment:
The extension failed to run because the `cheerio` package is used, but it is not listed as a dependency in `package.json`.